plugin/web/types.ts

```diff
@@ -0,0 +1,9 @@
+type actionType =
+  | "updateOverrides"
+  | "screenshot"
+  | "screenshot-save"
+  | "modal-open"
+  | "modal-close"
+  | "msgFromModal"
+  | "minimize";
```

Comment:
Sorry can you keep the `actionType` extension specific? To do that, you'll have to make PostMessageProps generic, so something like:
```
export type PostMessageProps<T> = { action: T; payload?: any };
```